e2e/test/iothub/service/RegistryE2ETests.cs

```diff
@@ -131,7 +131,7 @@ public async Task DevicesClient_AddDeviceWithTwinWithDeviceCapabilities()
 
         [LoggedTestMethod]
         [Timeout(TestTimeoutMilliseconds)]
-        public async Task DevicesClient_AddDevices2Async_Works()
```

Comment:
not sure what the number 2 in the test name stands for, so delete it

Reply:
There was a v1 API called AddDevices2Async (because there used to be an AddDevicesAsync and they wanted to rev it without breaking the existing API).